tests/erdos.gv

```diff
@@ -0,0 +1 @@
+digraph G {	graph [rankdir="LR"]	{ node [shape=box]	sum_squares_square [ label = "sum_squares_square" ];default_sum_squares_output_op [ label = "default_sum_squares_output_op" ];sum_squares_sum [ label = "sum_squares_sum" ];default_int2 [ label = "default_int2" ];default_int1 [ label = "default_int1" ];default_sum_squares_input_op [ label = "default_sum_squares_input_op" ];default_default_square [ label = "default_default_square" ];	}	{ edge [color="#cccccc"]	default_int2 -> default_sum_squares_input_op [ label = "" ];default_sum_squares_input_op -> sum_squares_square [ label = "" ];sum_squares_square -> sum_squares_sum [ label = "" ];sum_squares_sum -> default_sum_squares_output_op [ label = "" ];default_sum_squares_output_op -> default_default_square [ label = "" ];default_int1 -> default_sum_squares_input_op [ label = "" ];	}}
```

Comment:
Can you please remove this file from the commit? Same for all the other binary swp files.